pkg/cli/admin/inspect/inspect.go

```diff
@@ -101,6 +104,8 @@ func NewCmdInspect(streams genericclioptions.IOStreams, parentCommandPath string
 
 	cmd.Flags().StringVar(&o.destDir, "dest-dir", o.destDir, "Root directory used for storing all gathered cluster operator data. Defaults to $(PWD)/inspect.local.<rand>")
 	cmd.Flags().BoolVarP(&o.allNamespaces, "all-namespaces", "A", o.allNamespaces, "If present, list the requested object(s) across all namespaces. Namespace in current context is ignored even if specified with --namespace.")
+	cmd.Flags().StringVar(&o.since, "since", o.since, "An RFC3339 timestamp from which to show logs. If this value precedes the time a pod was started, only logs since the pod start will be returned.")
+	cmd.Flags().DurationVar(&o.sinceTime, "since-time", o.sinceTime, "Only return logs after a specific date (RFC3339). Defaults to all logs. Only one of sinceTime or sinceseconds may be used.")
```

Comment:
`since` should be `DurationVar`, `since-time` - string. You should parse the latter in `Complete` and return error before even starting doing anything.